dashboard/app/jobs/application_job.rb

```diff
@@ -44,10 +44,9 @@ def self.common_dimensions
         value: job_count,
         unit: 'Count',
         timestamp: Time.now,
-        dimensions: {
-          name: 'Environment',
-          value: CDO.rack_env
-        },
+        dimensions: [
+          {name: 'Environment', value: CDO.rack_env},
+        ]
```

Comment:
This actually supports arrays or single hashes, but our `includes_metrics` test tooling only supports array `dimensions`, so I'm taking the easy way out by wrapping this.